JetStreamDriver.js

```diff
@@ -994,9 +993,16 @@ class Benchmark {
         if (this._resourcesPromise)
             return this._resourcesPromise;
 
-        const filePromises = !isInBrowser ? this.plan.files.map((file) => fileLoader.load(file)) : [];
+        this.preloads = [];
+        this.blobs = [];
 
-        const promise = Promise.all(filePromises).then((texts) => {
+        if (isInBrowser) {
+            this._resourcesPromise = Promise.resolve();
```

Comment:
I haven't fully understood the (somewhat intertwined, complex) preload, resource loading code. At a high-level, why do we just early return here having added anything to `this.scripts`?